src/main/java/io/cdap/e2e/pages/locators/CdfControlCenterLocators.java

```diff
@@ -0,0 +1,167 @@
+  public static WebElement deleteIconSetPreferences;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='entity-card-header dataset']" +
+    "//*[@class='icon-svg icon-datasets entity-icon']")
+  public static WebElement datasetEntityIcon;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='header']//*[@class='icon-svg icon-datasets']")
+  public static WebElement datasetEntityDetailsPage;
+
+  @FindBy(how = How.XPATH, using = "(//*[@class='icon-svg icon-trash'])[1]")
+  public static WebElement datasetDeleteIcon;
+
+  @FindBy(how = How.XPATH, using = "(//*[@class='icon-svg icon-cut'])[1]")
+  public static WebElement truncateDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[@class='link-to-detail']")
+  public static WebElement viewDetailsTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Dataset')]")
+  public static WebElement navigateOnViewDetailsPageDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Back')]")
+  public static WebElement backTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//*[@class='icon-svg icon-close']")
+  public static WebElement closeTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying all entities, sorted by Newest')]")
+  public static WebElement datasetApplicationsArtifactsMessage;
+
+  public static By locateDropdownListItem(String option) {
+    return By.xpath("//input[@data-cy='" + option + "']");
+  }
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Schema')]")
+  public static WebElement schemaTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Programs')]")
+  public static WebElement programsTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='schema-tab']")
+  public static WebElement schemaTabDatasetEntityPage;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='program-tab clearfix']")
+  public static WebElement programsTabDatasetEntityPage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Newest')]")
+  public static WebElement newestOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Oldest')]")
+  public static WebElement oldestOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Z - A')]")
+  public static WebElement zToAOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by A - Z')]")
+  public static WebElement aToZOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//div[@id='filter-tooltip-target-id']")
+  public static WebElement sortDropdown;
+
+  public static WebElement selectSortDropdownValue(String option) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//*[contains(text(),'" + option + "')]"));
+  }
```

Comment:
do not locate elements using text content.

Reply:
This also dont have any data-testid , so we need to use this to choose the option for sorting